File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.go
func kidsWithCandies(candies []int, extraCandies int) []bool {
    most := candies[0]
    for _, candy := range candies {
        if candy > most {
            most = candy
        }
    }

    result := make([]bool, len(candies))

    for i, candy := range candies {
        result[i] = candy + extraCandies >= most
    }

    return result
}

File: 1528-kids-with-the-greatest-number-of-candies/kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, candies: List[int], extraCandies: int) -> List[bool]:
        most = max(candies)
        result = []

        for candy in candies:
            if extraCandies + candy >= most:
                result.append(True)
            else:
                result.append(False)

        return result